src/main/java/io/xspec/maven/xspecMavenPlugin/resolver/ResolverS9.java

```diff
@@ -50,7 +50,7 @@
  * Saxon URI Resolver should never be used.
  * @author cmarchand
  */
-public class Resolver implements javax.xml.transform.URIResolver, EntityResolver {
+public class ResolverS9 implements javax.xml.transform.URIResolver, EntityResolver {
```

Comment:
Well it's kind of a relict of when I had another implementation based on the Saxon11 `ResourceResolver`. That would need more refactoring though, so I abandoned it after noticing that I could get it to work with `URIResolver`.

So at this point, this rename could well be undone.

In general my suggestion would be to always name the `ResolverDecorator` after the first Saxon version that is supported by it. Hence the S9 in the name right now. The one based on the `ResourceResolver` could then be named S11 (since that is the first version to support the ResourceResolver).